docs/installation/installation-using-docker-application-setup.md

```diff
@@ -3,51 +3,15 @@
 This guide expects that you have already set up your Docker environment.
 If you haven't already done that check the [Installation Using Docker](installation-using-docker.md).
 
-## 1. Setup the microservice for product search
-Installing of Shopsys Framework from a project-base requires manual cloning of the [repository of the microservice](https://github.com/shopsys/microservice-product-search).
-
-Clone the repository inside a separate directory and configure the new container `microservice-product-search` in your `docker-compose.yml` according to the updated templates in [project-base/docker/conf](https://github.com/shopsys/project-base/tree/master/docker/conf).
-
-Edit the path `./microservices/product-search` of the new container in such a way so it leads to the directory with the cloned microservice. 
-
-*Note: Soon, a need of cloning of the microservice will be removed and the microservice will be downloaded automatically when installing the Shopsys Framework.*
-
-*Note: There can be differences in the configuration on different OS.
-For example the configuration for mac requires changes in `docker-sync.yml`, uid and guid can be different across platforms, etc. .
-It should be similar to the `php-fpm`'s configuration.*
-
-*Note: If you have edited the docker-compose.yml, you should re-run the command `docker-compose --force-recreate -d`.*
-
-### 1.1. Connect into terminal of the Docker container
-```
-docker exec -it shopsys-framework-microservice-product-search sh
-```
-### 1.2. Install dependencies
-```
-composer install
-```
-### 1.3. Run the server
-Since the microservice acts as a fully independent unit, it needs a separate web server.
-In the current version of this experimental microservice, the PHP's built-in Web Server is used.
-
-You can start the server by executing the command:
-```
-php bin/console server:run *:8000
-```
-
-In this moment the microservice is ready for the requests processing.
-
-*Note: You can switch the running process to run in the background by pressing `CTRL+Z` and executing `bg`.*
-
-## 2. Setup the Shopsys Framework application
+## 1. Setup the Shopsys Framework application
 Now that the Docker environment is prepared and the product search microservice is up and running, we can setup the application itself.
```

Comment:
In this sentence there is reference to running product search microservice as it was individual from other containers. But it isnt now, right? So i would change this sentence to "Now that the Docker environment is prepared and running, we can setup the application itself."